elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -106,3 +112,34 @@ $LOCAL-VARIABLES: (
 :host([hidden]) {
   display: none;
 }
+
+/// ===========================================================================
+/// CONTEXTS: DARK & SATURATED
+/// ===========================================================================
+/// In dark & saturated contexts, we override the local color vars
+
+
+:host {
+  @at-root #{&}([pfe-variant="earth"][on="dark"]) {
+    --pfe-tabs__panel--BackgroundColor: #{pfe-var(surface--darkest)};
+  }
+}
+
+:host {
+  @at-root #{&}([pfe-variant="earth"][on="saturated"]) {
+    --pfe-tabs__panel--BackgroundColor: #{pfe-var(surface--lightest)};
+
+      ::slotted(*) {
+        color: pfe-var(text);
```

Comment:
I think you can set this on the host and not have to assign it to slotted.  It should cascade.

Reply:
you are correct 👍